gemini-k8s-cluster

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Convenient script to run gemini on cluster
+#
+# N.B:
+# make sure kubectl has the same version as server
+
+# variables
+declare -r DB_HOST='scylladb'
+declare -r SPARK_MASTER='spark://p-spark-master:7077'
+
+# create pod if not exists
+if ! kubectl get pods | grep gemini > /dev/null; then
+  kubectl create -f ./k8s/gemini-pod.yaml
```

Comment:
should we take care of errors that could occur?

Reply:
fixed

Reply:
If I understood you when the [`kubectl create`](https://github.com/src-d/gemini/pull/35/files#diff-87240ec88599237cfcde88be7dd27039R14) exits with `0`, there is no feasible way to know whether the pod will be created, or it won't, or it will take forever. If so, I'm ok with the infinite `while` loop.

Reply:
yes. `kubectl create` just creates task to create container in kubernetes. If it can't reach k8s for example it will exist with error. But otherwise it will just notify "ok, now we will try to create". And creation itself can fail because of many reasons:
- wrong configuration
- docker hub is unavailable
- no resources on cluster
- broken image
- million other reasons

it's possible to check for some other statuses besides `Success`. Like `CrashLoopBackoff` but I don't want to handle all possible statuses here. They can be different depends on k8s version and you will have to look what went wrong manually anyway.